book/tutorials/database/3_forming_queries.ipynb

```diff
@@ -104,12 +104,7 @@
   },
   {
    "cell_type": "markdown",
-   "metadata": {
-    "tags": [
-     "nbsphinx-gallery",
-     "nbsphinx-thumbnail"
-    ]
-   },
+   "metadata": {},
```

Comment:
not sure why the metadata changed in all the database notebooks

Reply:
I think when I was cleaning notebooks I ran the clean *.ipynb command in the wrong directory. I reverted all these files back to the commit before I did that.